app/javascript/packages/stylelint-config/package.json

```diff
@@ -17,12 +17,11 @@
   },
   "homepage": "https://github.com/18f/identity-idp",
   "dependencies": {
-    "stylelint-config-prettier": "^9.0.3",
```

Comment:
The package is essentially deprecated now, since the major change of Stylelint v15 is to deprecate style-related rules.

See "Note" at the top of: https://www.npmjs.com/package/stylelint-config-prettier